docs/modules/ROOT/pages/references/composite/dbaas/exoscale/redis.adoc

```diff
@@ -0,0 +1,8 @@
+= pkg.appcat.composite.dbaas.exoscale.redis
+
+The parent key for all of the following parameters is`pkg.appcat.composite.dbaas.exoscale.redis`.
```

Comment:
```suggestion
The parent key for all of the following parameters is `pkg.appcat.composite.dbaas.exoscale.redis`.
```

Reply:
nice catch :+1: updated the otherr files too/